etc/schema.json

```diff
@@ -1034,6 +1038,22 @@
                                     }
                                 },
                                 "additionalProperties": false
+                            },
+                            "sshfp": {
+                                "type": "object",
+                                "description": "A Secure Shell fingerprint is used to verify the system’s authenticity",
```

Comment:
`description` and `type` are a bit inconsistently ordered here. I learned from the reviews on my such PR that it is preferred to have `description` as the first item.

Reply:
got it, I'll reorder and submit a new PR